Library/Homebrew/cask/lib/hbc/cli/style.rb

```diff
@@ -11,7 +11,7 @@ def self.help
 
       def run
         install_rubocop
-        system "rubocop", *rubocop_args, "--", *cask_paths
+        system({ "XDG_CACHE_HOME" => HOMEBREW_CACHE }, "rubocop", *rubocop_args, "--", *cask_paths)
```

Comment:
I tried to keep this as locally scoped as possible. No strong opinion if it would be better to set this globally.